### PR TITLE
constructor sonarcloud fixes, magic numbers and refactors

### PR DESCRIPTION
**Pull request by GodsonLeigh**

### CODE REVIEW

File: app/config/database.js
Comment (JungaBunga) [inline app/config/database.js:1]: I would probably given a more context name for variable, such as portNumber or similiar

File: jest.setup.js
Comment (JungaBunga) [inline jest.setup.js:1]: here is the same, I would give context such as timeoutSeconds (if it uses seconds as parameter)

**Review by JungaBunga — COMMENTED** [review]
Clear implementation

File: jest.setup.js
Comment (GodsonLeigh) [reply jest.setup.js:1]: For the port number, I agree, however I am trying to keep the variables consistent here and in cases like this where we are using a number of them on the same page  where they could potentially be reused for different functions

File: app/config/database.js
Comment (GodsonLeigh) [reply app/config/database.js:1]: Agreed, in this situation it makes sense. I'll amend

**Review by JungaBunga — APPROVED** [review]
image

**Review by JungaBunga — APPROVED** [review]
image

**Pull request merged**